src/lamplib/src/genny/tasks/yaml_linter.py

```diff
@@ -56,7 +56,12 @@ def lint(self, genny_repo_root: str, lint_format: bool):
 
         if lint_format:
             config_file_path = path.join(genny_repo_root, ".yamllint")
-            yamllint_argv = ["--strict", "--config-file", config_file_path] + all_yamls
+            yamllint_argv = (
```

Comment:
why change this line from `all_yamls` ... it the same thing?

Reply:
`all_yamls` is a list of all yaml files, which is really long. This change is to display the top level directories only instead of the entire list. IMO, this improves the readability of the screen output.

Reply:
yeah, that makes sense as long as ```yamllint.cli.run(yamllint_argv)``` matches the same files (and it would be surprising if it didn't).

Maybe move the code from line 36 to line 65,  ```all_yamls``` is only used in the log line now?

Reply:
done